.github/workflows/soak-testing.yml

```diff
@@ -99,8 +99,6 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
So these were for a user due to the previous OIDC limitations, but now we can remove the user and these long term creds right?

Reply:
Use it was for a user, correct!

Well originally we could not use OIDC because [the token only lasted for 5 minutes](https://github.com/aws-observability/aws-otel-java-instrumentation/pull/90). OIDC is able to refresh the sessional credentials on every AWS call, but the Soak Tests might wait up to 10 minutes to make another AWS call.

The hope is that now, with a 1 hour expiry, that is plenty of time for the Soak Tests to refresh credentials every time it polls the alarms (every 10 minutes) to see if a spike occurred. If the Soak Tests ever poll longer than that then we would have a bigger problem...

However, I have not tested this on my soak-testing-demo fork and was counting on just merging it and fixing it if it breaks 😅... I can move this file to a separate PR if we want to test it on just the Integration Tests first 🙂